test/opencensus/plug/trace_test.exs

```diff
@@ -45,14 +38,13 @@ defmodule Opencensus.Plug.TraceTest do
     test "after response the parent span continues", %{conn: conn} do
       ctx = :oc_trace.start_span("span", :undefined, %{})
 
-      header =
-        ctx
-        |> :oc_span_ctx_header.encode()
-        |> List.to_string()
+      headers =
+        for {k, v} <- :oc_propagation_http_tracecontext.to_headers(ctx) do
+          {k, List.to_string(v)}
+        end
 
       _ =
-        conn
-        |> put_req_header(:oc_span_ctx_header.field_name(), header)
+        List.foldl(headers, conn, fn {k, v}, acc -> Plug.Conn.put_req_header(acc, k, v) end)
```

Comment:
It is common practice in Elixir to start pipeline with raw value. Also I think that `Enum.reduce/3` would suit more this case.

Reply:
Ok, updating. And will change to use `Enum.reduce/3`, is there an actual different between the 2?

Reply:
`Enum.reduce/3` is probably a little bit slower, but on the other hand it is more idiomatic.